### PR TITLE
Device modal

### LINKED ISSUE #30: Device modal
Add inline form as in criteria in Issues.

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

## Related issues

Resolve #30

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -82,8 +82,9 @@ model User {
 
   role UserRole @default(USER)
 
-  createdAt DateTime @default(dbgenerated("timezone('utc'::text, now())")) @db.Timestamp()
-  updatedAt DateTime @default(dbgenerated("timezone('utc'::text, now())")) @updatedAt
+  createdAt  DateTime     @default(dbgenerated("timezone('utc'::text, now())")) @db.Timestamp()
+  updatedAt  DateTime     @default(dbgenerated("timezone('utc'::text, now())")) @updatedAt
+  UserDevice UserDevice[]
```
Comment (tsumo) [inline prisma/schema.prisma:87]: Should be named `devices` and placed higher, with the other relations

File: src/components/AddDeviceToUserForm/AddDeviceToUserForm.i18n/ru.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "Device": "Девайс",
+    "id": "",
```
Comment (tsumo) [inline src/components/AddDeviceToUserForm/AddDeviceToUserForm.i18n/ru.json:3]: Doesn't need translation

File: src/components/AddDeviceToUserForm/AddDeviceToUserForm.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+import { Device } from 'prisma/prisma-client';
+import { IconPlusCircleSolid } from '@taskany/icons';
+
+import { trpc } from '../../trpc/trpcClient';
+import { InlineTrigger } from '../InlineTrigger';
+import { useDeviceMutations } from '../../modules/deviceHooks';
+
+import { tr } from './AddDeviceToUserForm.i18n';
+
+const StyledInput = styled(Input)`
```
Comment (tsumo) [inline src/components/AddDeviceToUserForm/AddDeviceToUserForm.tsx:14]: Let's ask in chat, maybe if bricks already has an input styled like this

**Review by tsumo — CHANGES_REQUESTED** [review]

**Review by 9teen90nine — APPROVED** [review]

File: src/modules/modules.i18n/ru.json [hunk 4]
```diff
@@ -20,5 +20,6 @@
     "Cannot move archived group": "Нельзя перемещать аархивированную группу",
     "Cannot move group into archived group": "Нельзя переместить группу в заархивированную группу",
     "Cannot edit archived group": "Нельзя редактировать заархивированную группу",
-    "Group is archived": "Группа заархивирована"
+    "Group is archived": "Группа заархивирована",
+    "Choose a device": ""
```
Comment (tsumo) [inline src/modules/modules.i18n/ru.json:24]: Needs translation

**Review by tsumo — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**